aptos-move/framework/src/aptos-natives.bpl

```diff
@@ -0,0 +1,22 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// ==================================================================================
+// Native object::exists_at
+
+var  #0_$memory: $Memory #0;
```

Comment:
What is this?

Reply:
It's a temporary hack to make a memory space for the type parameter `#0`. However, the reason for this may be worth thinking about.

Monomorphization doesn't seem to create a memory space for a generic type that which the key-ability. It is because it has been not needed until now. We don't allow using the global storage operators (`exists`, `global`) for the generic type. However, with `object::exists_at`, one can do a global storage operator (`exists`) with a generic type. For example,
```
    public fun address_to_object<T: key>(object: address): Object<T> {
        assert!(exists<ObjectCore>(object), error::not_found(EOBJECT_DOES_NOT_EXIST));
        assert!(exists_at<T>(object), error::not_found(ERESOURCE_DOES_NOT_EXIST));
        Object<T>{ inner: object }
    }
```

Reply:
I see. I guess the declarations of `var  #0_$memory: $Memory #0;` should happen alongside the declarations of `#0` then as there might be `#1`, `#2`, etc.